tools/google-cloud-support-slackbot/google_cloud_support_slackbot.py

```diff
@@ -0,0 +1,811 @@
+    user_id : str
+        the Slack user_id of the user who submitted the request. Used to send ephemeral
+        messages to the user
+    reason : str
+        reason for the escalation. Must be a value of either REASON_UNSPECIFIED, 
+        RESOLUTION_TIME, TECHNICAL_EXPERTISE, or BUSINESS_IMPACT
+    justification : str
+        user submitted string justifying the need for an escalation
+    user_name : str
+        Slack user_name of the user that ran the command. Appended to the end of the
+        justification to identify who submitted the escalation, otherwise all escalations 
+        will show as coming from the case creator
+    """
+    
+    client.chat_postEphemeral(channel=channel_id, user=user_id, text="Your request is processing ... ")
+    parent = get_parent(case)
+    if parent == 'Case not found':
+        client.chat_postEphemeral(channel=channel_id, user=user_id, text=f"Case {case} could not be found in your org. If this case was recently created, please give the system 60 seconds to fetch it. Otherwise, double check your case number or confirm the org being tracked with your Slack admin.")
+    else:
+        signed_justification = justification + '\n *Sent by {} via Google Cloud Support Slack bot'.format(user_name)
+        body = {
+                'escalation' : {
+                    'reason': reason,
+                    'justification': signed_justification
+                    }
+                }
+        escalation_mask = ['escalation.reason', 'escalation.justification']
+        #try:
+        req = support_service.cases().escalate(name=parent, body=body)
+        req.execute()
+        client.chat_postEphemeral(channel=channel_id, user=user_id, text=f"You have escalated case {case}")
+        #except Exception as e:
+            #client.chat_postEphemeral(channel=channel_id, user=user_id, text="Your attempt to escalate may have failed. Please contact your account team or try again later.")
+        client.chat_postEphemeral(channel=channel_id, user=user_id, text=f"You have escalated case {case}")
+
+
+
+def stop_tracking(channel_id, channel_name, case, user_id):
+    """
+    Remove a case from the list of tracked Google Cloud support cases
+
+    Parameters
+    ----------
+    channel_id : str
+        unique string used to idenify a Slack channel. Used to send messages to the channel
+    channel_name : str
+        user designated channel name. For users to understand where their cases are being 
+        tracked in Slack
+    case : str
+        unique id of the case
+    user_id : str
+        the Slack user_id of the user who submitted the request. Used to send ephemeral
+        messages to the user
+    """
+    
+    exists = False
+    for tc in tracked_cases:
+        if tc['channel_id'] == channel_id and tc['case'] == case:
+            tracked_cases.remove(tc)
+            
+            try:
+                file_overwrite(tracked_cases_file, tracked_cases)
+            except Exception as e:
```

Comment:
What specific exception are you capturing here? Please narrow it down to the specific situations that you are trying to handle, this is too broad.